pkg/controllers/deprovisioning/singlemachineconsolidation.go

```diff
@@ -70,9 +70,9 @@ func (c *SingleMachineConsolidation) ComputeCommand(ctx context.Context, candida
 			return Command{}, fmt.Errorf("command is no longer valid, %s", cmd)
 		}
 
-		if cmd.action == actionReplace || cmd.action == actionDelete {
+		if cmd.Action() != NoOpAction {
```

Comment:
Would this ever be the case because of L60?

Reply:
No it shouldn't! Good catch.